client/contracts/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "sharding",
```

Comment:
I think this package is probably from anther project

Reply:
Yes, it was taken from the test project I've done during the interview process :) I will fix that